docs/examples/helpbutton/endGuideElement.js

```diff
@@ -0,0 +1,22 @@
+// @flow strict
+import { type Node } from 'react';
+import { Checkbox, Flex, HelpButton, Text } from 'gestalt';
+
+export default function Example(): Node {
+  return (
+    <Flex height="100%" justifyContent="center" alignItems="center" gap={1}>
+      <Checkbox
+        labelDisplay="hidden"
+        label="Created pins checkbox"
+        id="created-pins"
+        onChange={() => {}}
+      />
+      <Text>Pins created in the last 30 days</Text>
+      <HelpButton
+        accessibilityLabel="Click to learn more about last 30 days of created pins"
+        accessibilityPopoverLabel="Expanded information about about last 30 days of created pins"
```

Comment:
Expanded information about the last 30 days of created pins

Reply:
Done